### ISSUE TITLE
SomePart_Edit Shape definition in placement.info file from an active theme is not working

### ISSUE THREAD

**Issue by owen26** [OrchardCMS/Orchard#5347]
this is probably not a bug as described in this post:
http://stackoverflow.com/questions/11417280/orchard-cms-1-4-2-edit-view-for-my-custom-content-type-doesnt-show-content-pa

but I think there are definitely something that can be improved.

Occasionally I work on small projects to build regular company websites. Orchard is my first choice as always. When I develop such small project I prefer to create a Theme with all the unique business logics in it including special Controllers and Parts just for this particular website. These parts won't be useful for other projects. And no other special Modules are necessary for such small website.

This gives me at least one big advantage that everything necessary for this project is packed in to one decent Theme package. No need for additional Modules. And it's also good for source code management as I don't want to put all Orchard CMS codes in git for a tiny website project. 

But the problem is I can not make my custom Part editor visible on admin pages as placement.info definition is not working there. What I can do currently is put my placment.info file into TheAdmin theme folder. But that's not a very good solution.

So in conclusion, I think it would be a good idea to make Edit Shape placement in current Theme working (or take priority) over TheAdmin Theme and other modules.

**Comment by sfmskywalker** [comment]
If we do this, then it seems to make sense to also look for shape alternate view templates that live in themes. E.g. I have a theme that implements custom Layout elements. These elements are rendered using the "Design" display type, but the shape alternates currently are not found because they live in a theme instead of a module.

**Comment by owen26** [comment]
For the same reason if I want override tinyMCE behaviour by creating Body-Html.Editor.html file in my active theme, it's not gonna work.

Currently I have to put it in TheAdmin Theme or any of my custom Module's View folder.

**Comment by owen26** [comment]
I really hope this can be improved in the near future so that I no longer need to create both Theme and Modules for my tiny projects. Just one Theme would be sufficient.
